src/utils/constants.rs

```diff
@@ -0,0 +1,11 @@
+use alloy_primitives::{address, Address};
```

Comment:
These are really development constants, we will want to add the actual testnet/mainnet constants as well. They likely should come from the SDK.

Reply:
CC @Tjemmmic